imdb_django/management/commands/import_principals.py

```diff
@@ -0,0 +1,78 @@
+# myapp/management/commands/import_principals_data.py
+
+import csv
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from imdb_django.models import Name, Title, TitlePrincipal
+from helper import log_info  # Import the log_info function
```

Comment:
Let's remove the comment, it is stating the fact which is very obvious

Please update all files, where required